src/components/modal/base/modal.scss

```diff
@@ -4,7 +4,7 @@
 .modal-content {
     position: relative;
     margin: 3.75rem auto;
-    border-radius: 1rem;
+    border-radius: 0.5rem;
```

Comment:
/cc @carljbowman

Reply:
@benjiwheeler - Thinking we should keep the border-radius 1rem for now. We can circle back on modifying the global level style. It's a little uncertain how this might impact other modals on the site.

It's okay that the design spec is out of sync with WWW.

Reply:
keeping 1rem for now. Chris, let's talk about that option after the commit.